tests/topotests/bgp_recursive_route_ebgp_multi_hop/test_bgp_recursive_route_ebgp_multi_hop.py

```diff
@@ -257,10 +257,10 @@ def test_recursive_routes_iBGP_peer_p1(request):
     )
     dut = "r2"
     create_interface_in_kernel(
-        tgen, dut, "lo", "40.40.40.50", netmask="255.255.255.0", create=True
+        tgen, dut, "lo10", "40.40.40.50", netmask="255.255.255.0", create=True
     )
     create_interface_in_kernel(
-        tgen, dut, "lo", "40:40::40:50", netmask="120", create=True
+        tgen, dut, "lo10", "40:40::40:50", netmask="120", create=True
```

Comment:
As explained in the commit, when `create_interface_in_kernel` runs on an existing interface, like `lo` in this case, it flushes all the existing IP addresses on this interface, including the ones configured by FRR (zebra). Then during the test there's a place where the daemon config is getting read again, and the previous IP addresses are restored. But this doesn't happen anymore after NB conversion, because the NB infra sees that the config is already applied and doesn't reapply it again, and the test fails. By changing the interface name, we don't flush the existing IPs from `lo` and the test runs fine again.

Actually, this is a behavior change that we should probably mention in the release notes. This applies not only to zebra, but to any daemons and commands converted to NB. Previously, any configuration command that is repeated, would just do the same thing again that it did on the first run. Now, a repeated command just does nothing.

Reply:
Not necessarily a behaviorial change imo.  I know of other cases where we go already configured nothing to do here.

Reply:
Sure, there are a lot of commands that do nothing if already configured. But it's not a general rule and depends on the implementation of a specific command. With NB conversion it becomes a general rule.